frontend/src/store/wallet.ts

```diff
@@ -631,19 +631,13 @@ const hasSetPublicKeysLegacy = async (name: string, provider: Provider) => {
 
 // Helper method to check if user has registered public keys in the StealthKeyRegistry
 async function getRegisteredStealthKeys(account: string, provider: Provider) {
```

Comment:
As I thought about it, and saw how slow it was if both the subgraph query and registry contract query failed.. I felt that because we are falling back from one to the other, that in the case of them both failing, there is likely some kind of a network problem and a retry probably isn't helpful.